File: ASCII2NUM/C#/ASCII2NUM.cs
using System;

namespace ASCII2NUM
{
    class MainClass
    {
        public static void Main()
        {
            Console.WriteLine("Enter a character: ");
            char character;
            character = (char)Console.Read();

            Console.WriteLine(); //Just for printing a new line

            int charvalue = (int)character;

            Console.WriteLine("The character " + character + "'s ASCII value is " + (char)charvalue);
        }
    }
}

File: ASCII2NUM/Java/ASCII2NUM.java
import java.io.IOException;

/**
 *
 * @author disablegraphics
 */
public class ASCII2NUM {

    /**
     * @param args the command line arguments
     * @throws java.io.IOException
     */
    public static void main(String[] args) throws IOException {
        // TODO code application logic here
        System.out.println("Please insert a character: ");
        char character;
        character = (char) System.in.read();
        int charvalue;
        charvalue = (int)character;
        System.out.println("The ASCII code of " + character + " equals to: " + charvalue);
    }
    
}
